app/src/main/java/com/adammcneilly/pokedex/displaymodels/ItemDisplayModel.kt

```diff
@@ -0,0 +1,26 @@
+package com.adammcneilly.pokedex.displaymodels
+
+import com.adammcneilly.pokedex.models.Item
+import com.adammcneilly.pokedex.util.ItemImageURLGenerator
+
+data class ItemDisplayModel(
+    val id: String,
+    val name: String,
+    val image: ImageDisplayModel,
+    val description: String,
+) {
+    constructor(item: Item) : this(
+        id = item.id,
+        name = item.parseName(),
+        image = ImageDisplayModel.Remote(
+            ItemImageURLGenerator.generate(item.name),
+        ),
+        description = item.description,
+    )
+}
+
+private fun Item.parseName(): String {
+    return name.split("-").joinToString(" ") { item ->
+        item.replaceFirstChar(Char::uppercaseChar)
+    }
+}
```

Comment:
This is a leaky abstraction because it's based on the apollo API we're using. We might want to pull this out of this layer.